nilearn/tests/test_extmath.py

```diff
@@ -12,7 +12,7 @@
 def test_fast_abs_percentile():
     data = np.arange(1, 100)
     for p in range(10, 100, 10):
-        yield nose.tools.assert_equal, fast_abs_percentile(data, p-1), p
+        yield nose.tools.assert_equal, fast_abs_percentile(data, p - 1), p
```

Comment:
so I discovered something the other day, PEP8 doesn't require you to have a space around binary operators. See [this](https://www.python.org/dev/peps/pep-0008/#other-recommendations) for example.

Interestingly, autopep8 fixes something that pep8 doesn't see as a violation ...